tests/Operation/CreateEncryptedCollectionFunctionalTest.php

```diff
@@ -5,7 +5,7 @@
 use MongoDB\BSON\Binary;
 use MongoDB\BSON\Document;
 use MongoDB\Client;
-use MongoDB\ClientEncryption;
+use MongoDB\Driver\ClientEncryption;
```

Comment:
Any reason this wasn't caught before?

Reply:
I think psalm would've caught this, but we haven't enabled it for tests. I'll enable it in PHPLIB-1119 and add all current test errors to the baseline.